src/main/java/edu/tamu/iiif/service/AbstractManifestService.java

```diff
@@ -165,10 +165,22 @@ protected Model getRdfModel(String url) throws NotFoundException {
     }
 
     protected String getRdf(String url) throws NotFoundException {
-        Optional<String> rdf = Optional.ofNullable(restTemplate.getForObject(url, String.class));
-        if (rdf.isPresent()) {
-            logger.debug("RDF for {}: \n{}\n", url, rdf.get());
-            return rdf.get();
+        try {
+            if (logger.isDebugEnabled()) {
+                logger.info("Requesting RDF for {}", url);
+            }
+            Optional<String> rdf = Optional.ofNullable(restTemplate.getForObject(url, String.class));
+            if (rdf.isPresent()) {
+                if (logger.isDebugEnabled()) {
```

Comment:
I would think this `logger.isDebugEnabled()` is not necessary when `logger.debug()` is called. Is this not the case?

Reply:
I do not have a definitive answer, but I have switched between using isDebugEnabled as thinking the call to debug is not necessary and incur more cost than the isDebugEnabled call and condition.